business_rules/utils.py

```diff
@@ -120,6 +124,28 @@ def check_params_valid_for_method(method, given_params, method_type_name):
             ', '.join(invalid_params), method_type_name, method.__name__))
 
 
+def check_for_default_value_for_missing_params(missing_params, method_params):
+    """
+    :param missing_params: Params missing from Rule
+    :param method_params: Params defined on method, which could have default value for missing param
+    [{
+     'label': 'release voucher from template',
+     'name': 'voucher_template_id',
+     'fieldType': 'numeric',
+     'defaultValue': 123
+    },
+    ...
+    ]
+    :return Updated missing parameters:
+    """
+    if method_params:
+        for param in method_params:
+            if param['name'] in missing_params and param.get('defaultValue'):
```

Comment:
`param.get('defaultValue')` in the condition is dangerous in this case:
```python
In [1]: bool(0)
Out[1]: False

In [2]: bool('')
Out[2]: False

In [3]: bool(None)
Out[3]: False
```
All above cases could be a default value

Reply:
Added a check for empty string and 0. If defaultValue=None here it means either it was set in Action as None or it wasn't set at all. In the first case Platform already has default value of none for parameters and in the second one we just ignore it.